scripts/get-suppliers-with-copyable-services.py

```diff
@@ -5,19 +5,29 @@
 in tests.
 
 Usage:
-    scripts/get-suppliers-with-copyable-services.py <stage> <new-framework> <copy-from-framework>
+    scripts/get-suppliers-with-copyable-services.py <stage> <new-framework> <copy-from-framework> [options]
+
+Options:
+    --limit=<limit>             Limit the number of suppliers to find (default is 1000)
```

Comment:
```suggestion
    --limit=<limit>             Limit the number of suppliers to find [default: 1000]
```

Nit: docopt can handle defaults itself: https://github.com/docopt/docopt#option-descriptions-format

Reply:
I did not know I could do this, thanks for the link it was very useful